### PR TITLE
Blockchain c1w6 - Anonimity

### PR DESCRIPTION
**Pull request by mihaiberq**
References #557

### CODE REVIEW

**Comment by enki-hq** [comment]
404 error making request GET https://api.github.com/repos/enkidevs/curriculum/contents/blockchain/fundamentals/identity-and-anonimity/anonymity.md: "Not Found"

File: blockchain/fundamentals/anonimity/anonymity.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Regarding Bitcoin, we can talk about a *pseudoanonymity*. Say you were the recipient of two transactions in 
+the past, two UTXOs of 1 and 2 bitcoins, respectively. Now, if you are looking to sell more than 2 bitcoins at the same time, you would have to consume both UTXOs.
+
+Unless you are the owner of the addresses, you wouldn't be able to digital sign the transaction combining them. Hence, we can guess that they belong to the same user/wallet. Your identity wasn't revealed, but your transactions can now be tracked.
+
+---
+## Practice
+
+Complete the following saying:
```
Comment (lizTheDeveloper) [inline blockchain/fundamentals/anonimity/anonymity.md]: Probably add a more realistic question here

File: blockchain/fundamentals/anonimity/how-to-de-anonymize-bitcoin.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+And depending on where they found out about the exact value of the transaction, they can even know your real identity. A barista that shows you the coffeeshop's public address can also link your identity to the transaction. By donating bitcoins and sharing the donation confirmation to incentivize others to do the same, you are also revealing your blockchain identity.
+
+Why should you care though? Well, chances are you won't have that many bitcoins anyway. If you made a good investment and have a large sum of money in your bank account, there's a slim chance of anyone knowing it unless you told someone. But your Bitcoin address is public and will always be public.
+
+If you decide to buy more bitcoins because you belive it is the coin of the future, what stops others from trying to steal your private key? Attention to details and precaution go a long way. To make the whole tracking process harder, *mixing services*, or simply mixers, appeared.
```
Comment (lizTheDeveloper) [inline blockchain/fundamentals/anonimity/how-to-de-anonymize-bitcoin.md]: Add some comments here about 
- mixers not protecting you fully
- how mixers actually work

File: blockchain/fundamentals/anonimity/how-to-de-anonymize-bitcoin.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+links:
+  - '[Bitcoin transactions are not that anonymous.](https://www.technologyreview.com/s/608716/bitcoin-transactions-arent-as-anonymous-as-everyone-hoped/)'
+
+---
+## Content
+
+In the previous insight, we mentioned a specific way of linking transactions to an online identity: joint payments. Most of the times, however, a single redeemed UTXO is enough to track someone if the transacted value is specific enough.
+
+A bitcoin transaction can represent up to 8 digits after the decimal points (`0.000000001 bitcoin` is called a `satoshi`), so the distribution space is large. If someone is to know the exact value of your transaction and a timeframe in which you had intiated the transaction, there's a chance they could track your address.
```
Comment (lizTheDeveloper) [inline blockchain/fundamentals/anonimity/how-to-de-anonymize-bitcoin.md]: Note here what the odds of someone exchanging the name exact decimal value in an hour, given the maximum number of transactions the bitcoin network can process per hour, and how many hours of time need to elapse before that value gets close enough to 1 to de-anonymize you

**Comment by enki-hq** [comment]
Rule insight:standards failed for files: 
  - blockchain/fundamentals/anonimity/anonymity.md
  - blockchain/fundamentals/anonimity/how-to-de-anonymize-bitcoin.md
  - blockchain/fundamentals/anonimity/mixing.md

**Pull request merged**